internal/services/mysql/mysql_flexible_server_resource_test.go

```diff
@@ -548,7 +548,7 @@ resource "azurerm_mysql_flexible_server" "test" {
   location               = azurerm_resource_group.test.location
   administrator_login    = "_admin_Terraform_892123456789312"
   administrator_password = "QAZwsx123"
-  sku_name               = "B_Standard_B1s"
+  sku_name               = "GP_Standard_D2ds_v4"
```

Comment:
Why do we need to update the `sku_name` for the basic config if this property can be updated?

Reply:
@stephybun you are right, I have updated the `sku_name` for the basic config. Thanks!